### PR TITLE
chore: update vite app configs for vue-wagmi example

### PR DESCRIPTION
**Pull request by enesozturk**
# Description

Please include a brief summary of the change.

## Type of change

- [x] Chore (non-breaking change that addresses non-functional tasks, maintenance, or code quality improvements)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Associated Issues

For Linear issues: Closes APKT-xxx
For GH issues: closes #...

# Showcase (Optional)

If there is a UI change include the screenshots with before and after state.
If new feature is being introduced, include the link to demo recording.

# Checklist

- [ ] Code in this PR is covered by automated tests (Unit tests, E2E tests)
- [x] My changes generate no new warnings
- [x] I have reviewed my own code
- [ ] I have filled out all required sections
- [x] I have tested my changes on the preview link
- [ ] Approver of this PR confirms that the changes are tested on the preview link

### CODE REVIEW

File: examples/vue-ethers5/src/App.vue [hunk 1]
```diff
@@ -1,84 +1,139 @@
...
+    networkState.value = state
+  })
+
+  modal.subscribeTheme(state => {
+    themeState.value = state
+    document.body.className = state.themeMode
+  })
+
+  modal.subscribeWalletInfo(state => {
+    // @ts-ignore
```
Comment (magiziz) [inline examples/vue-ethers5/src/App.vue:65]: Needed ?

File: examples/vue-ethers5/src/main.ts [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import { createApp } from 'vue'
+// @ts-ignore
```
Comment (magiziz) [inline examples/vue-ethers5/src/main.ts:2]: Same here ?

File: examples/vue-solana/src/App.vue [hunk 3]
```diff
@@ -36,64 +27,128 @@ const solanaAdapter = new SolanaAdapter({
...
+    networkState.value = state
+  })
+
+  modal.subscribeTheme(state => {
+    themeState.value = state
+    document.body.className = state.themeMode
+  })
+
+  modal.subscribeWalletInfo(state => {
+    // @ts-ignore
```
Comment (magiziz) [inline examples/vue-solana/src/App.vue:80]: Same here ?

**Review by magiziz — APPROVED** [review]
lgtm. Just left a few question comments on `@ts-ignore`

File: examples/vue-ethers5/src/App.vue (hunk 1 above)
Comment (enesozturk) [reply examples/vue-ethers5/src/App.vue:65]: On the Vue the TS extension was not working fine, didn't want to spend time resolving it. It's sample app related thing, not the AppKit itself, so safe to ignore for now

**Pull request merged**